README.md

```diff
@@ -757,6 +757,8 @@ You can enable live form data validation by passing a `liveValidate` prop to the
 
 Be warned that this is an expensive strategy, with possibly strong impact on performances.
 
+Completely disable the validation, you can use prop `noValidate` set in `true`.
```

Comment:
*To disable validation entirely, you can set Form's `noValidate` prop to `true`.*